### PR TITLE
I hate old guns?

### PR DESCRIPTION
**Pull request by Draganfrukts**
## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
balance: rebalanced something
imageadd: added some icons and images

/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
valve, add this

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 1]
```diff
@@ -268,42 +268,43 @@
...
+/obj/item/ammo_box/magazine/internal/boltaction/quartermaster
+	name = "repeater magazine tube"
 	desc = "How did you get it out?"
-	ammo_type = /obj/item/ammo_casing/a762
-	caliber = CALIBER_A762
-	max_ammo = 8
+	ammo_type = /obj/item/ammo_casing/a357
+	caliber = CALIBER_357
+	max_ammo = 12
 	multiload = TRUE
```
Comment (Hatterhat) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:308]: might wanna drop multiload on this unless you want .357 speedloaders to work for this? i don't actually remember how it works but thems the breaks

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 2]
```diff
@@ -268,42 +268,43 @@
...
-/obj/item/ammo_box/magazine/internal/boltaction/bubba
-	name = "sportiv extended internal magazine"
+/obj/item/ammo_box/magazine/internal/boltaction/quartermaster
+	name = "repeater magazine tube"
 	desc = "How did you get it out?"
-	ammo_type = /obj/item/ammo_casing/a762
-	caliber = CALIBER_A762
-	max_ammo = 8
+	ammo_type = /obj/item/ammo_casing/a357
+	caliber = CALIBER_357
```
Comment (Iamgoofball) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:306]: undocumented balance change

File: modular_skyrat/modules/aesthetics/guns/code/guns.dm [hunk 3]
```diff
@@ -268,42 +268,43 @@
...
+	max_ammo = 12
```
Comment (Iamgoofball) [inline modular_skyrat/modules/aesthetics/guns/code/guns.dm:307]: undocumented balance change

**Comment by Gandalf2k15** [comment]
reopen when u start working on it

**Pull request closed without merging**